Cargo.lock

```diff
@@ -2727,9 +2727,9 @@ dependencies = [
 
 [[package]]
 name = "h2"
-version = "0.3.16"
+version = "0.3.18"
```

Comment:
Thanks for the patch.

Can you split it out as a separate PR and tag CSUB-408 in the title.

Reply:
I dropped that commit and launched the dependabot PR